install_config/upgrading/automated_upgrades.adoc

```diff
@@ -329,6 +329,91 @@ $ ansible-playbook -i <path/to/inventory/file> \
 See xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Manging
 Nodes] for more on node labels.
 
+
+[[upgrade-hooks]]
+=== Customizing Upgrades With Ansible Hooks
```

Comment:
I moved the subsection down in the order a little bit vs https://github.com/openshift/openshift-docs/pull/3643.